components/table/README.md

```diff
@@ -522,3 +523,12 @@ The `d2l-table-controls` component can be placed in the `d2l-table-wrapper`'s `c
 | `no-sticky` | Boolean | Disables sticky positioning for the controls |
 | `select-all-pages-allowed` | Boolean | Whether all pages can be selected |
 <!-- docs: end hidden content -->
+
+## Accessibility
+
+The `d2l-table-wrapper` simply wraps an HTML table in order to add styles and functionality to that table. As such, it is important to use proper table markup. Recommendations are available in [this tutorial](https://www.w3.org/WAI/tutorials/tables/). As mentioned in the Best Practices section of this document, it is important to specify column and row headings so data is meaningful to screen reader users.
```

Comment:
I think the second sentence could be its own paragraph or bullet point? And could drop the "As mentioned in Best Practices" part. Someting like this:

> Make sure to include [column and row headings](https://www.w3.org/WAI/tutorials/tables/) so data is meaningful to screen reader users